scripts/codegen.sh

```diff
@@ -18,14 +18,11 @@ fi
 # Regenerate code
 ./gradlew -p codegen/sdk-codegen build
 
-# Delete the current autogenerated SDK code
-rm -rf release/
```

Comment:
if there is "left over", we need to fix root cause than duct taping by removing release/services dir. 

bigger workspace is an issue and by adding tests we are amplifying it. I don't have solution for it. Potentially making multiple Packages.swift for each service but solution to this problem is not related to this PR.

Reply:
Deleting previous generated code isn't duct taping, it should be an integral part of this process because it prevents artifacts from previous generation of the project from compiling if they're not explicitly overwritten.

If you don't want to delete artifacts from previous code generation before staging the new ones, then that means if we ever remove an AWS service or even a source file within a service, we have to come up with a way to get rid of it if we don't delete all of the previous ones.

If you don't want to have to write `rm -rf ...` into several Github workflows and build scripts, then we should do it at the start of the `stageSdks` Gradle task.

Reply:
Sorry, I don't agree with deletion. Specially in a case where we a failure in the codegen (if skipped) for a (more services) would lead to a removal of service.

If we remove a file/service it must be manual as accidental removal of something will lead to catastrophic outcomes. 

For a legit service removal, we must delete the model and delete the generated code manually.